docs/source/llms/langchain/index.rst

```diff
@@ -156,110 +164,39 @@ exploring these more advanced use cases.
     </section>
 
 
-`Detailed Documentation <guide/index.html>`_
---------------------------------------------
-
-To learn more about the details of the MLflow LangChain flavor, read the detailed guide below.
 
-.. raw:: html
+Logging models from Code
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-    <a href="guide/index.html" class="download-btn">View the Comprehensive Guide</a>
+Since MLflow 2.12.2, MLflow introduced the ability to log LangChain models directly from a code definition.
 
-.. toctree::
-    :maxdepth: 1
-    :hidden:
+The feature provides several benefits to manage LangChain models:
 
-    guide/index.rst
-
-FAQ
```

Comment:
The diff went crazy while I just moved up the "How can I log my chain from code?" question to a dedicated section. There is no changes in other part of the FAQ. I believe it's much easier to review via the preview doc.